src/backend/agent.js

```diff
@@ -87,6 +87,10 @@ export default class Agent extends EventEmitter {
     bridge.addListener('startProfiling', this.startProfiling);
     bridge.addListener('stopInspectingDOM', this.stopInspectingDOM);
     bridge.addListener('stopProfiling', this.stopProfiling);
+    bridge.addListener(
+      'syncSelectionFromBrowserTools',
+      this.syncSelectionFromBrowserTools
```

Comment:
trivial nit: How do you feel about replacing "_BrowserTools_" with "_ElementsPanel_" ? Seems more specific.

Reply:
Wanted to make it somehow clear that this isn’t taking about any of our own panels.

Reply:
Yeah, I guess "browser tools" just means nothing to me.

Reply:
Maybe “native elements tab”?

Reply:
👍